samples/greengrass/basic_discovery/main.cpp

```diff
@@ -53,20 +43,15 @@ int main(int argc, char *argv[])
     const char **const_argv = (const char **)argv;
     cmdUtils.SendArguments(const_argv, const_argv + argc);
 
-    if (cmdUtils.HasCommand("help"))
-    {
-        cmdUtils.PrintHelp();
-        exit(-1);
-    }
-    certificatePath = cmdUtils.GetCommandRequired("cert");
-    keyPath = cmdUtils.GetCommandRequired("key");
-    thingName = cmdUtils.GetCommandRequired("thing_name");
-    caFile = cmdUtils.GetCommandOrDefault("ca_file", caFile);
-    region = cmdUtils.GetCommandOrDefault("region", region);
-    topic = cmdUtils.GetCommandOrDefault("topic", topic);
-    mode = cmdUtils.GetCommandOrDefault("mode", mode);
-    message = cmdUtils.GetCommandOrDefault("message", message);
-    proxyHost = cmdUtils.GetCommandOrDefault("proxy_host", proxyHost);
+    String certificatePath = cmdUtils.GetCommandRequired("cert");
+    String thingName = cmdUtils.GetCommandRequired("thing_name");
+    String caFile = cmdUtils.GetCommandOrDefault("ca_file", "");
+    String keyPath = cmdUtils.GetCommandOrDefault("key", "");
+    String region = cmdUtils.GetCommandOrDefault("region", "us-east-1");
```

Comment:
remove the "us-east-1" default
AWS has been trying to move away from having this as a hardcoded default